### PR TITLE
Post-mortem for 2019-02-18 upload issue

### PR DESCRIPTION
**Pull request by hem-brahmbhatt**
Doesn't need to have a perfect form, but if incorrect statements are made they should be corrected.

The corrective actions need to be agreed upon by the team.

### CODE REVIEW

**Review by diversemix — APPROVED** [review]

**Review by Cooryd — APPROVED** [review]

**Review by jyenes — APPROVED** [review]

**Comment by diversemix** [comment]
@SemraSh ... Are you okay to approve these changes? ( you are the last important one ;) )

File: docs/incidents/2019-02-18-file-upload.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+
+## Stabilization Steps
+
+- Revert the change which introduced the issue in production
+- Rerun the pipeline to fix the failure on CI for the revert fix
+- Commit version was checked on the HTML of the application pages in Production, and verified that it was correct.
+
+## Impact
+
+- Caused a bad UX where user would not know whether there file was uploading, and could have concluded that the app was broken.
```
Comment (giorgiosironi) [inline docs/incidents/2019-02-18-file-upload.md:76]: needs Mean Time To Discover and Mean Time To Repair, see https://github.com/elifesciences/tech-team/blob/master/incidents/2017-09-08-Journal-stack-disappearing.md#impact
should be quick subtractions (`discovery - introduction` and `repair - discovery`)

**Review by SemraSh — APPROVED** [review]

File: docs/incidents/2019-02-18-file-upload.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
+# 2019-02-18
+
+**Incident Leader: **
```
Comment (BlueReZZ) [inline docs/incidents/2019-02-18-file-upload.md]: Missing an incident leader - needs to be a single individual

**Review by BlueReZZ — APPROVED** [review]

File: docs/incidents/2019-02-18-file-upload.md (hunk 1 above)
Comment (SemraSh) [reply docs/incidents/2019-02-18-file-upload.md]: @SemraSh

Comment (SemraSh) [reply docs/incidents/2019-02-18-file-upload.md]: @SemraSh

**Review by giorgiosironi — APPROVED** [review]

**Pull request merged**